app/v2/notifications/post_notifications.py

```diff
@@ -291,6 +298,14 @@ def process_sms_or_email_notification(*, form, notification_type, api_key, templ
         )
         persist_scheduled_notification(notification.id, form["scheduled_for"])
 
+    elif current_app.config["FF_REDIS_BATCH_SAVING"] and not simulated:
+        redis_queue = RedisQueue()
+        if notification_type == SMS_TYPE:
+            redis_queue.publish(encrypted_notification_data)
```

Comment:
there will be two separate redis queues here?

Reply:
yeah, exactly.